src/content-scripts/canvas.js

```diff
@@ -0,0 +1,75 @@
+var canvas, ctx, flag = false,
+    prevX = 0,
+    currX = 0,
+    prevY = 0,
+    currY = 0,
+    dot_flag = false;
+
+var x = "black",
+    y = 2;
+
+var overlayPage = {
+  zIndex: 100,
+  position: 'absolute',
+  top: 0,
+  left: 0
+};
+
+$(function(){
+  
+  $('<canvas id="can"></canvas>')
+    .css(overlayPage)
+    .attr('width', document.body.scrollWidth) // sets to max width
+    .attr('height', document.body.scrollHeight) // sets to max height
+    .on('mousemove', function(e){findxy('move', e)})
+    .on('mousedown', function(e){findxy('down', e)})
+    .on('mouseup', function(e){findxy('up', e)})
+    .on('mouseout', function(e){ findxy('out', e)})
+    .appendTo('body')
+
+  canvas = document.getElementById('can');
+  ctx = canvas.getContext("2d");
+  w = canvas.width;
+  h = canvas.height;
+})
+
+function draw() {
+  ctx.beginPath();
+  ctx.moveTo(prevX+pageXOffset, prevY+pageYOffset);
+  ctx.lineTo(currX+pageXOffset, currY+pageYOffset);
+  ctx.strokeStyle = x;
+  ctx.lineWidth = y;
+  ctx.stroke();
+  ctx.closePath();
+}
+
+function findxy(res, e) { 
+  if (res == 'down') {
+    prevX = currX;
+    prevY = currY;
+    currX = e.clientX - canvas.offsetLeft;
+    currY = e.clientY - canvas.offsetTop;
+
+    flag = true;
+    dot_flag = true;
+    if (dot_flag) {
```

Comment:
I've removed the dot_flag since its unnecessary. 

Just to notet the other flag is necessary to distinguish drawing mousemove events and normal mousemove events.